### PR TITLE
Feature/null check

### PR DESCRIPTION
**Pull request by jason-fox**
Potential fix for https://github.com/telefonicaid/iotagent-json/issues/523

### CODE REVIEW

File: lib/services/ngsi/entities-NGSI-LD.js [hunk 1]
```diff
@@ -68,93 +68,98 @@ function valueOfOrNull(value) {
```
Comment (fgalan) [inline lib/services/ngsi/entities-NGSI-LD.js:68]: I understand this PR doesn't need to add any entry to CNR file given the existing "Add: basic NGSI-LD support as experimental feature (#842)" already covers it, right?

Comment (jason-fox) [reply lib/services/ngsi/entities-NGSI-LD.js:68]: Correct.

Comment (fgalan) [reply lib/services/ngsi/entities-NGSI-LD.js:68]: NTC

**Comment by jason-fox** [comment]
I've completed updating the test cases. The remaining error is just a **Coveralls** rounding error.

**Comment by fgalan** [comment]
> I've completed updating the test cases. The remaining error is just a Coveralls rounding error.

Maybe coveralls can be configured with some "tolerance" degree? I mean, for instance, if the difference is less than 0.05% don't mark it as fail.

**Review by fgalan — APPROVED** [review]
LGTM

**Comment by jason-fox** [comment]
> > I've completed updating the test cases. The remaining error is just a Coveralls rounding error.
> 
> Maybe coveralls can be configured with some "tolerance" degree? I mean, for instance, if the difference is less than 0.05% don't mark it as fail.

You can do this - just read https://docs.coveralls.io/coveralls-notifications

Go to https://coveralls.io/github//telefonicaid/iotagent-node-lib/settings

Screenshot 2021-01-26 at 17 37 25

**Pull request merged**